### DISCUSSION TITLE
Getting access to database

### DISCUSSION THREAD

**Discussion by kshitij01042002** [narayan954/dummygram#105] [General]
How can i get access to the firebase?. You haven't mentioned anything about it in installation steps

**Comment by narayan954** [comment]
@kshitij01042002 I am assuming you want access to the Database.
Api key gives the access to the DB. Currently it's present in the firebase.js file, but in future the API key will be stored as an environment variable and while setting up the project, you might have to get your own API key and documentations will be update with that.

**Reply by kshitij01042002** [reply]
Can i just change the Api key and connect project with my fb database? Will it be fine or are there anythings I have to do prior to this?

**Reply by narayan954** [reply]
> Can i just change the Api key and connect project with my fb database? Will it be fine or are there anythings I have to do prior to this?

yes, you can generate your own api key and work with that, though the db will be blank and you'll have to create some dummy post to see the structure.
